administration/intro.adoc

```diff
@@ -332,4 +332,55 @@ to fix it, delete manually finalizer from the resource.
 ----
 $ kubectl -n kubevirt patch kv kubevirt --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
 ----
+
+Note: If you find your namespace is still hanging in terminationstate this script will delete all objects inside of a namespace.
+[source,bash]
+```
+# the namespace of kubevirt installation
+export namespace=kubevirt
+export labels=("operator.kubevirt.io" "operator.cdi.kubevirt.io" "kubevirt.io" "cdi.kubevirt.io")
+export namespaces=(default ${namespace} "<other-namespaces that has kubevirst resources>")
+
+kubectl -n ${namespace} delete kv kubevirt
+kubectl -n ${namespace} patch kv kubevirt --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
+kubectl get vmis --all-namespaces -o=custom-columns=NAME:.metadata.name,NAMESPACE:.metadata.namespace,FINALIZERS:.metadata.finalizers --no-headers | grep foregroundDeleteVirtualMachine | while read p; do
+    arr=($p)
+    name="${arr[0]}"
+    namespace="${arr[1]}"
+    kubectl patch vmi $name -n $namespace --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
+done
+
+for i in ${namespaces[@]}; do
+    for label in ${labels[@]}; do
+        kubectl -n ${i} delete deployment -l ${label}
```

Comment:
this is quite repetitive. I think that
```
for kind in deployment ds rs ....; do
```
would improve this. However, this code is tightly related to what hco does; I would suspect this code should sit in hco and not here.

Reply:
It is repetitive. The point is to get every single possible kubernetes object